recipes/ostaphep/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
Can you remove the extra white space from this file. Having one blank line is nice for making it more readable but there two or more has the opposite effect.